stdlib/multiprocessing/managers.pyi

```diff
@@ -124,8 +124,14 @@ class SyncManager(BaseManager):
     def Semaphore(self, value: Any = ...) -> threading.Semaphore: ...
     def Array(self, typecode: Any, sequence: Sequence[_T]) -> Sequence[_T]: ...
     def Value(self, typecode: Any, value: _T) -> ValueProxy[_T]: ...
-    def dict(self, sequence: Mapping[_KT, _VT] = ...) -> _dict[_KT, _VT]: ...
-    def list(self, sequence: Sequence[_T] = ...) -> _list[_T]: ...
+    @overload
+    def dict(self, sequence: Mapping[_KT, _VT]) -> _dict[_KT, _VT]: ...
+    @overload
+    def dict(self) -> _dict[Any, Any]: ...
```

Comment:
This is another instance of the issue @erictraut found in https://github.com/microsoft/pyright/issues/3501#issuecomment-1135455934, though here it's on a function, not a constructor.

Reply:
Yeah, empty collections are tough :(